### PR TITLE
added set rollback duration documentation

### PR DESCRIPTION
**Pull request by hellt**
Accompanying PR for https://github.com/openconfig/gnmi/pull/164

/cc @kidiyoor

### CODE REVIEW

File: rpc/gnmi/gnmi-commit-confirmed.md [hunk 1]
```diff
@@ -103,8 +117,20 @@ If the server is not waiting for confirmation or if the value doesn’t match th
...
+
 Cancellation can be issued by providing `CancelRequest` as action in the extension. The value of `id`
 should be equivalent to the id of the on-going commit.
 
 If the server is not waiting for cancellation or if the value doesn’t match the on-going commit
 then FAILED_PRECONDITION or INVALID_ARGUMENT error is returned respectively.
+
+### 3.2.4 Set Rollback Duration
+
+When a commit is on-going, a new rollback duration can be set by providing `SetRollbackDurationRequest`
```
Comment (kidiyoor) [inline rpc/gnmi/gnmi-commit-confirmed.md:129]: Could you make it clear if the existing rollback counter will be reset or if the new duration value value will be added on to the on-going counter ?

**Comment by kidiyoor** [comment]
LGTM

File: rpc/gnmi/gnmi-commit-confirmed.md (hunk 1 above)
Comment (hellt) [reply rpc/gnmi/gnmi-commit-confirmed.md]: done in 19cd325

Comment (LimeHat) [reply rpc/gnmi/gnmi-commit-confirmed.md]: Perhaps a similar comment, but I think we need a bit more clarity here.

Can you please explicitly specify the "starting point" of the new timer (do we count the duration from the initial commit timestamp, or from 0 after receiving `SetRollbackDurationRequest`)?

Comment (hellt) [reply rpc/gnmi/gnmi-commit-confirmed.md]: done in [0c1d8f4](https://github.com/openconfig/reference/pull/203/commits/0c1d8f4216e0f51b4994f0053305e7933696c35f)

**Review by kidiyoor — APPROVED** [review]

**Comment by hellt** [comment]
@dplore since the proto for this feature has been merged, can we merge also the doc part?

**Review by dplore — APPROVED** [review]

**Pull request merged**